examples/google-esp-integration/with_sdk_v3/public/stylesheets/app.css

```diff
@@ -0,0 +1,106 @@
+body {
```

Comment:
I know these css files were copied from the example-sites repo - but is it worth addressing some css nits in this PR? There are some formatting issues, some missing semicolons, some `0px` and some unnecessary decimals e.g. `1.0px`. Autoformatter in vscode will fix most of them automatically.

Reply:
same question applies to the server_only `app.css` too